OpenNefia.Content/Prototypes/Protos/Protos.Tag.cs

```diff
@@ -11,6 +11,9 @@ public static class Tag
             public static readonly TagPrototypeId DungeonStairsDelving   = new($"Elona.{nameof(DungeonStairsDelving)}");
             public static readonly TagPrototypeId DungeonStairsSurfacing = new($"Elona.{nameof(DungeonStairsSurfacing)}");
 
+            public static readonly TagPrototypeId Lomias                 = new($"Elona.{nameof(Lomias)}");
```

Comment:
Maybe `UniqueCharaLomias` to make the purpose clearer.

Reply:
changed in 5da0180